.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,85 @@
+        
+        Before submitting a new bug/issue, please check the links below to see if there is a solution or question posted there already:
+         - `remix-graphql` Issues tab: https://github.com/thomasheyenbrock/remix-graphql/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc
+         - `remix-graphql` closed issues tab: https://github.com/thomasheyenbrock/remix-graphql/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed
+         - `remix-graphql` Discussions tab: https://github.com/thomasheyenbrock/remix-graphql/discussions
+
+        The more information you fill in, the better the community can help you.
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: Provide a clear and concise description of the challenge you are running into.
+    validations:
+      required: true
+  - type: input
+    id: link
+    attributes:
+      label: Your Example Website or App
+      description: |
+        Which website or app were you using when the bug happened?
+        Note:
+        - Your bug will may get fixed much faster if we can run your code and it doesn't have dependencies other than the Remix Auth npm package.
+        - To create a shareable code example you can use Stackblitz (https://stackblitz.com/). Please no localhost URLs.
+        - Please read these tips for providing a minimal example: https://stackoverflow.com/help/mcve.
+      placeholder: |
+        e.g. https://stackblitz.com/edit/...... OR Github Repo
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to Reproduce the Bug or Issue
+      description: Describe the steps we have to take to reproduce the behavior.
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behavior
+      description: Provide a clear and concise description of what you expected to happen.
+      placeholder: |
+        As a user, I expected ___ behavior but i am seeing ___
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots_or_videos
+    attributes:
+      label: Screenshots or Videos
+      description: |
+        If applicable, add screenshots or a video to help explain your problem.
+        For more information on the supported file image/file types and the file size limits, please refer
+        to the following link: https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/attaching-files
+      placeholder: |
+        You can drag your video or image files inside of this editor ↓
+  - type: textarea
+    id: platform
+    attributes:
+      label: Platform
+      value: |
+        - OS: [e.g. macOS, Windows, Linux]
+        - Browser: [e.g. Chrome, Safari, Firefox]
+        - Version: [e.g. 91.1]
```

Comment:
Not sure if this list contains the right bullet points 🤔 I believe bugs in this library much rather come down to the Node.js version or the deploy target ("vanilla" Remix Server, Vercel, CF Worker, etc.). Since none of the code here ends up in the browser, specifying a browser doesn't really make much sense here. OS might be relevant though, I'd leave that in.

Reply:
Good point - I will update to reflect this

Reply:
Updated 🙏 
- https://github.com/cliffordfajardo/remix-graphql/commit/482e1f04ab6d6ac3466031f79f4c722e5237e8d3

Reply:
Nice! Should we remove the generic `Platform` textarea now?

Reply:
whoops forgot to remove - good catch 🦅 
 @thomasheyenbrock - updated & removed: https://github.com/thomasheyenbrock/remix-graphql/pull/4/commits/95007ba35e62e5be4142e4f28c3cf99eb49a6644